features/manage_category.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Manage Category
+  In order to maintain typo blog
+  As an admin
+  I want to view and manage categories
+
+  Scenario: Categories List
```

Comment:
I think you'll need to update this to include some page navigation, but I think that's what you meant when  you mentioned updating to run the cucumber tests